vars/sectionDeployToAKS.groovy

```diff
@@ -143,6 +143,13 @@ def call(params) {
                 }
               }
             }
+            if (environment == env.nonProdName) {
+              stageWithAgent("Uninstall release ${environment}", product) {
```

Comment:
looking good, have you tested it somewhere? i.e link to build?

Reply:
You need something like : `new Environment(env).nonProdName`

Reply:
Hey, I've fixed that and added the links for the test builds